tests/integration_tests/functional/test_api.py

```diff
@@ -82,10 +82,8 @@ def test_drive_io_engine(uvm_plain):
 
     test_microvm.start()
 
-    # Execute a simple command to check that the guest booted successfully.
-    rc, _, stderr = test_microvm.ssh.run("true")
-    assert rc == 0
-    assert stderr == ""
+    # Check that the guest booted successfully.
```

Comment:
nit: here and below: I don't think such comments are necessary now

Reply:
indeed. will drop them!